packages/server/package.json

```diff
@@ -9,7 +9,7 @@
     "postbuild": "tsc --build tsconfig.build.json",
     "start": "node dist/bin.js",
     "test": "jest",
-    "test:pg": "../pg/with-test-db.sh jest",
+    "test:pg": "../pg/with-test-db.sh yarn test",
```

Comment:
no issues. but when I want to drill in on a certain test file & was hardcoding it in the `test` command & this just makes it so I don't have to hardcode twice 😛

Reply:
Ahhh got it!  I just hadn't run into it since I've been tacking the file onto the end of the command, e.g.:
```
yarn workspace @atproto/server test:pg packages/server/tests/views/profile.test.ts
```

Reply:
ah that's not a bad idea